app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -115,6 +115,11 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         if (toShow) {
             navUrlInput.requestFocus()
             updateNavigationButtons()
+        } else {
+            // #393: Youtube doesn't refocus properly, so refresh
+            if (navUrlInput.text.contains("youtube.com/tv")) {
```

Comment:
This might be wrong - what if the user types in "youtube.com/tv" and closes the drawer without loading? The UI state doesn't always match the model state. For this reason, I think we should delegate to BrowserFragment's – it knows the true url.